internal/controller/linodemachine_controller.go

```diff
@@ -169,6 +170,51 @@ func (r *LinodeMachineReconciler) Reconcile(ctx context.Context, req ctrl.Reques
 	return r.reconcile(ctx, log, machineScope)
 }
 
+func (r *LinodeMachineReconciler) reconcilePause(ctx context.Context, logger logr.Logger, machineScope *scope.MachineScope) error {
+	// Pause
+	isPaused, conditionChanged, err := paused.EnsurePausedCondition(ctx, machineScope.Client, machineScope.Cluster, machineScope.LinodeMachine)
+
+	if err == nil && !isPaused && !conditionChanged {
```

Comment:
do you know what the error case is here where it could return an error but that doesn't matter?